core/tee/tee_svc_cryp.c

```diff
@@ -1332,11 +1332,14 @@ static TEE_Result copy_in_attrs(struct user_ta_ctx *utc,
 {
 	TEE_Result res;
 	uint32_t n;
+	size_t size;
```

Comment:
init to something (and maybe to 2 above also)

Reply:
OK for size, as for the others why do them and not all the other ones in the same file, it's a different thing I prefer not to mix with fixes.

Reply:
ok